<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DateStartEndWidget</class>
 <widget class="QWidget" name="DateStartEndWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>45</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QDateTimeEdit" name="dateStart">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>to</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDateTimeEdit" name="dateEnd">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
